development/nautobot_config.py

```diff
@@ -5,7 +5,18 @@
 from nautobot.core.settings import *
 from nautobot.core.settings_funcs import is_truthy, parse_redis_connection
 
+
+#
+# Misc. settings
+#
+
 ALLOWED_HOSTS = os.getenv("NAUTOBOT_ALLOWED_HOSTS", "").split(" ")
+HIDE_RESTRICTED_UI = os.getenv("HIDE_RESTRICTED_UI", False)
```

Comment:
At some point we should change this environment variable to `NAUTOBOT_HIDE_RESTRICTED_UI` for consistency, but that's obviously a possibly breaking change.

Reply:
Yeah we should!